packages/packagers/js/src/prelude.js

```diff
@@ -6,8 +6,7 @@
 // anything defined in a previous bundle is accessed via the
 // orig method which is the require for previous bundles
 
-// eslint-disable-next-line no-global-assign
-parcelRequire = function(modules, cache, entry, globalName) {
+(function(modules, cache, entry, globalName) {
   // Save the require from previous bundle to this closure if any
   var previousRequire = typeof parcelRequire === 'function' && parcelRequire;
```

Comment:
Whoops, the global object would be needed here as well (`global.parcelRequire`)

Reply:
yeah. Probably best just to make a local within our closure and use that throughout. Will clean up some of the repetition later too.